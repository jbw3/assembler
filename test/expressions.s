# expressions

PARS = (2 + 2) * 10
BITS = ~(0x18 >> 3)

start:

    addi r0, PARS
    addi r0, SIZE
    addi r0, BITS

    # infinite loop
    nop
    jmp here - 2

    nop

end:
SIZE = end - start
