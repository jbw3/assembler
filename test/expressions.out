b028
b00e
b0fc
b002
0000
c008
0000
